### DISCUSSION TITLE
Ways to add text in the .css file

### DISCUSSION THREAD

**Discussion by Leopard-CC** [community/community#36215] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm making my learning website and facing an issue
I want to know ways to add text in the .css file
Thanks

**Comment by jfullstackdev** [comment]
why, what's the problem of editing `.css` file ? 

you can also have 

https://jsfiddle.net/

no further setup, just paste your code, only for plain web without server side services

**Comment by agiledev31** [comment, marked as answer]
You can use pseudo element to add text in the css file.
For example 
```
.Text::before {
      content: "This is the text added in the css file";
}
```
For more details, please check [link](https://stackoverflow.com/questions/2741312/using-css-to-insert-text)
Hope this helpful to your problem.
